en/docs/deploy/deploy-api/exposing-apis-via-custom-hostnames.md

```diff
@@ -0,0 +1,159 @@
+# Exposing APIs via Custom Hostnames
+
+Virtual hosts (VHosts) enable you to expose APIs using specific hostnames that correspond to a Gateway environment. Each Gateway environment will have a minimum of one VHost. Admin users can manage Gateways by creating, updating, and deleting Gateway environments via the Admin Portal. Each Gateway environment will contain the details of each of the VHosts, which are the custom hostnames, that are applicable to that Gateway environment. As a result, API publishers can use the VHosts to group their APIs by selecting a virtual host when deploying an API to Gateway environment(s). In addition, application developers can access APIs by using the defined access URLs of the virtual host.
```

Comment:
@renuka-fernando , please create a folder named "includes" in the en/docs/ folder and add 3 blank MD files with the following names. The purpose of this is to reuse content within the exposing-apis-via-custom-hostnames.md.

prerequisites-apim.md
assign-custom-hostname.md
view-custom-hostname.md

Reply:
created files in `en/docs/includes/` directory, not in `en/docs/`